modules/configuration-api/src/main/java/org/apache/ignite/configuration/RootKey.java

```diff
@@ -38,8 +38,10 @@ public class RootKey<T extends ConfigurationTree<VIEWT, ?>, VIEWT> {
     /** Schema class for the root. */
     private final Class<?> schemaClass;
 
-    /** Marked with {@link InternalConfiguration}. */
-    private final boolean internal;
+    /** Marked with {@link ConfigurationExtension}. */
```

Comment:
Could you please mention that it is not only marked with `ConfigurationExtension` but also with flag internal=true.